otter/convergence/service.py

```diff
@@ -415,9 +419,9 @@ def __init__(self, log, dispatcher, buckets, partitioner_factory,
             create an :obj:`Partitioner` to distribute the buckets.
         """
         MultiService.__init__(self)
+        self.log = log.bind(system='converger')
```

Comment:
Oh. I am sorry I just realized that it _was_ otter_service earlier and I changed it to system. I thought it is new change you are suggesting. `otter_service` coming up :)